tests/pytorch/test_onnx_export.py

```diff
@@ -1193,9 +1190,9 @@ def test_export_core_attention(
 
 test_configs_multihead_attention = [
     #"use_mask, attn_mask_type"
-    (False,    "causal"),  # calls ScaledUpperTriangMaskedSoftmax
+    (False,    "no_mask"), # calls ScaledUpperTriangMaskedSoftmax
```

Comment:
We're repeating a test:
```suggestion
    (False,    "causal"),  # calls ScaledUpperTriangMaskedSoftmax
```

Reply:
Minor comment fix:
```suggestion
(False, "no_mask"),  # calls ScaledSoftmax
```